scribble-lib/scribble/acmart/.gitignore

```diff
@@ -0,0 +1 @@
+acmart.cls
```

Comment:
Q. is it ok to have this local `.gitignore`? Or should I move this line to the top-level `.gitignore`?